lib/string_validator.rb

```diff
@@ -0,0 +1,13 @@
+class StringValidator
+
+  def validate(string)
+    valid_brackets = []
+    brackets = { '{' => '}', '[' => ']', '(' => ')' }
+    string.each_char do |char|
```

Comment:
Is `each_char` more efficient, or is chars-ing the string and then mapping over it more efficient?